packages/remix-dev/vite/plugin.ts

```diff
@@ -682,7 +682,13 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
               url
             );
           },
-          fixStacktrace: vite.ssrFixStacktrace,
+          // If an error is caught within the request handler, let Vite fix the
+          // stack trace so it maps back to the actual source code
+          processRequestError: (error) => {
```

Comment:
Heads up that I've renamed this hook from `fixStacktrace` to `processRequestError`. We want to make sure that all dev server hooks are named in a Vite-agnostic way. As part of this decoupling from Vite, I've also made all of the hooks optional.

Reply:
I don't usually have a strong opinion (and good sense) on naming things, so please feel free and don't hesitate to tweak anything I named.

> decoupling from Vite

That sounds like a huge plan!
I'll keep in mind that when doing things in the future.

Reply:
We don't see any need for these specific hooks to be used outside of Vite. For now it's mainly just to avoid the weirdness of having references to Vite in `@remix-run/server-runtime` 😅

Reply:
I am getting errors such as:

```
Error: `line` must be greater than 0 (lines start at line 1)
```

and I am wondering if related to these changes.